### PR TITLE
[Enhancement] backend test

### PR DESCRIPTION
**Pull request by muhwagwa**
Backend Coverage 98% (didn't do mocking for external API - TMDB API)

Screen Shot 2021-12-09 at 8 47 23 PM
Screen Shot 2021-12-09 at 8 47 42 PM

### CODE REVIEW

File: backend/content/tests.py [hunk 1]
```diff
@@ -115,6 +127,22 @@ def test_content_detail(self):
...
+        /api/content/initialize/contents/
+        GET
+        """
+        response = self.logged_in_client.get('/api/content/initialize/contents/')
+        self.assertEqual(response.status_code, 201)
+
+    def test_genre_initialize(self):
+        """
+        /api/content/initialize/contents/
+        GET
```
Comment (cw10108) [inline backend/content/tests.py:141]: comment update needed

File: backend/content/tests.py [hunk 2]
```diff
@@ -163,6 +191,43 @@ def test_user_favorite_list_get(self):
         self.assertEqual(response.status_code, 404)
 
 
+    def test_content_favorite_get(self):
```
Comment (cw10108) [inline backend/content/tests.py:194]: Maybe we need 405 test case here

File: backend/ott/tests.py [hunk 3]
```diff
@@ -55,6 +55,32 @@ def setUp(self):
...
+        ott = Ott(
+            ott='Watcha',
+            membership='Basic',
+            max_people=1,
+            cost=7900,
+            image=tempfile.NamedTemporaryFile(suffix=".jpg").name)
+        ott.save()
+        self.assertEqual(str(ott), 'Watcha / Basic')
+
+    def test_ott_initialize(self):
```
Comment (cw10108) [inline backend/ott/tests.py:68]: need 405 case, same as above

**Review by cw10108 — COMMENTED** [review]
I think we need test for 405 case!

**Review by arclic — APPROVED** [review]

**Pull request merged**